lib/session.js

```diff
@@ -4,7 +4,7 @@ const path = require('path')
 
 // npm dependencies
 const session = require('express-session')
-const FileStore = require('session-file-store')(session)
+const FileStore = require('./session-file-store')
```

Comment:
small thing but this isn't an npm dependency any more so it's in the wrong list here, should be in local I think?